.packit.yaml

```diff
@@ -0,0 +1,217 @@
+      - copr-cli.spec
+    upstream_tag_template: copr-cli-{version}
+    actions:
+      post-upstream-clone:
+        - sh -c "wait-for-copr --owner $COPR_OWNER --project $COPR_PROJECT python-copr `git rev-parse --short HEAD`"
+
+  copr-frontend:
+    downstream_package_name: copr-frontend
+    upstream_package_name: copr-frontend
+    paths:
+      - ./frontend
+    specfile_path: copr-frontend.spec
+    files_to_sync:
+      - copr-frontend.spec
+    upstream_tag_template: copr-frontend-{version}
+    actions:
+      post-upstream-clone:
+        - sh -c "wait-for-copr --owner $COPR_OWNER --project $COPR_PROJECT python-copr `git rev-parse --short HEAD`"
+
+
+srpm_build_deps:
+  # The wait-for-copr tool is used to resolve waiting for dependent builds.
+  # (Comes from the packit/packit-stable Copr project that is already enabled in the environment.)
+  - wait-for-copr
+
+merge_pr_in_ci: False
+
+
+jobs:
+
+  # Pull request builds
+
+  - job: copr_build
+    packages:
+      - copr-backend
+      - copr-keygen
+      - copr-messaging
+      - copr-dist-git
+      - copr-frontend
+    trigger: pull_request
+    targets:
+      - fedora-all-x86_64
+    manual_trigger: True # needs to be triggered by the `/packit build` comment
+
+  - job: copr_build
+    packages:
+      - copr-selinux
+      - python-copr
+      - python-copr-common
+      - copr-cli
+    trigger: pull_request
+    targets:
+      - fedora-all-x86_64
+      - fedora-all-aarch64
+      - fedora-all-ppc64le
+      - epel-all-x86_64
+    manual_trigger: True # needs to be triggered by the `/packit build` comment
+
+  - job: copr_build
+    packages:
+      - copr-rpmbuild
+    trigger: pull_request
+    targets:
+      - fedora-all-x86_64
+      - fedora-all-aarch64
+      - fedora-all-ppc64le
+      - epel-9-x86_64
+      - epel-8-x86_64
+    manual_trigger: True # needs to be triggered by the `/packit build` comment
+
+  # Main commit builds:
+
+  - job: copr_build
+    packages:
+      - copr-backend
+      - copr-keygen
+      - copr-messaging
+      - copr-dist-git
+      - copr-frontend
+    trigger: commit
+    branch: main
+    targets:
+      - fedora-all-x86_64
+    owner: packit
+    project: copr
+    preserve_project: True
+    list_on_homepage: True
+
+  - job: copr_build
+    packages:
+      - copr-selinux
+      - python-copr
+      - python-copr-common
+      - copr-cli
+    trigger: commit
+    branch: main
+    targets:
+      - fedora-all-x86_64
+      - fedora-all-aarch64
+      - fedora-all-ppc64le
+      - epel-all-x86_64
+    owner: packit
+    project: copr
+    preserve_project: True
+    list_on_homepage: True
+
+  - job: copr_build
+    packages:
+      - copr-rpmbuild
+    trigger: commit
+    branch: main
+    targets:
+      - fedora-all-x86_64
+      - fedora-all-aarch64
+      - fedora-all-ppc64le
+      - epel-9-x86_64
+      - epel-8-x86_64
+    owner: packit
+    project: copr
+    preserve_project: True
+    list_on_homepage: True
```

Comment:
we already have listed https://copr.fedorainfracloud.org/coprs/g/copr/copr/ on homepage so this would make duplicity, so maybe unlist it for now and once we start to fully use packit then we can list it on HP again, WDYT?

Reply:
OK, makes sence.

Reply:
fixed